operator/src/test/sh/scaling/scalingActionTest.sh

```diff
@@ -31,7 +31,7 @@ test_get_domain_api_version() {
 
   result=$(get_domain_api_version)
 
-  assertEquals "Did not return expected api version" 'v8' "${result}"  
+  assertEquals "Did not return expected api version" 'v9' "${result}"
```

Comment:
I would suggest using a constant, here, to minimize the number of changes we'll need to make each time we upgrade the domain resource version.